tests/bluetooth/tester/overlay-le-audio.conf

```diff
@@ -102,6 +104,7 @@ CONFIG_BT_CSIP_SET_MEMBER=y
 
 # CSIP
 CONFIG_BT_CSIP_SET_COORDINATOR=y
+CONFIG_BT_CSIP_SET_COORDINATOR_MAX_CSIS_INSTANCES=3
```

Comment:
Why do we need multiple CSIS instances?

Reply:
This is needed for tests with three instances of lower tester.

Reply:
Ah, sorry, yes of course; I had read this as we had 3 CSIS instances as the set member. My bad :)